apps/web/src/components/layout/components/AuthLayout.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Box, createStyles } from '@mantine/core';
+import { ThemeProvider } from '../../../design-system/ThemeProvider';
+
+export default function AuthLayout({ children }: { children: React.ReactNode }) {
+  const { classes } = useStyles();
+
+  return (
+    <ThemeProvider>
+      <div>
+        <div className={classes.wrapper}>
+          <img src="/static/images/logo.png" alt="logo" style={{ maxWidth: 150 }} />
+        </div>
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            '@media (max-width: 1200px)': {
+              display: 'none',
+            },
+          }}>
+          <img
+            src="/static/images/notifications/notification_01.png"
+            alt="logo"
+            style={{ position: 'absolute', maxWidth: 400, left: '7%', top: '35%' }}
+          />
+          <img
+            src="/static/images/notifications/notification_02.png"
+            alt="logo"
+            style={{ position: 'absolute', left: '10%', top: '45%', maxWidth: 400 }}
+          />
+          <img
+            src="/static/images/notifications/notification_03.png"
+            alt="logo"
+            style={{ position: 'absolute', left: '7%', top: '55%', maxWidth: 400 }}
+          />
```

Comment:
Small suggestion:
I would wrap the 3 images in a single div and calculate it's total height with the 3 notifications.
I would than do this div to position absolute and top: calc(100% - **heightOfBlock** / 2);

And then all the images inside will just be margin Bottom spaced without absolute. 

In this case the distance won't differ when changing screen height like in this photo:
![CleanShot 2022-03-24 at 10 35 25](https://user-images.githubusercontent.com/8872447/159876415-06f4db1e-2802-489f-8faf-bb6437fa4e5f.png)